os/emscripten/pre.js

```diff
@@ -9,7 +9,13 @@ Module['websocket'] = { url: function(host, port, proto) {
      * If you run your own server you can setup your own WebSocket proxy in
      * front of it and let people connect to your server via the proxy. You
      * are best to add another "if" statement as above for this. */
-    return null;
+    if (location.protocol === 'https:') {
```

Comment:
Please add a newline before this statement; the comment has nothing to do with it now, so that might be confusing ;)